src/file-queue.ts

```diff
@@ -11,26 +11,32 @@ export function makeFileQueue() {
       return filesOrderedByDate.shift()
     },
     requeue: function (file: File) {
-      const index = binarySearch(filesOrderedByDate, fileEntry => {
-        return file === fileEntry
-          ? 0
-          : file.lastTouchedTimestamp - fileEntry.lastTouchedTimestamp ||
-              // Fallback to lexigraphical comparison (if timestamps match)
-              (file.filename > fileEntry.filename ? 1 : -1)
-      })
+      this.remove(file)
+      this.queue(file)
```

Comment:
Define a local constant `out` and return that directly. Replace `this` with `out[method]`.